src/include/migraphx/verify_args.hpp

```diff
@@ -31,11 +31,15 @@
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 
-MIGRAPHX_EXPORT
-bool verify_args(const std::string& name,
-                 const argument& ref_arg,
-                 const argument& target_arg,
-                 double tolerance = 80);
+MIGRAPHX_EXPORT bool verify_args_with_threshold(const std::string& name,
```

Comment:
This should be named `verify_args` and the other overload can be named `verify_args_with_tolerance`.

Reply:
Done.